flake.nix

```diff
@@ -13,6 +13,7 @@
   };
   inputs.parser-util.url = "github:flox/parser-util/v0";
   inputs.pkgdb.url = "github:flox/pkgdb";
+  inputs.sqlite3pp.url = "github:aakropotkin/sqlite3pp";
```

Comment:
Can we instead vendor this into the flox repository or organization?

Reply:
I don't see why it has to be included here at all since it's a transitive dependency.

It's already set to follow the Nixpkgs input for pkgd ( so are all other deps there ) so just ride the train and follow the pkgdb set.

Keep in mind that the pkgdb test suite is run against the inputs in its lockfile.
So if you override those you become responsible for rerunning that suite with the modified input

Reply:
True, can just grab it via transitive... wilco